adapters/tlsio_mbedtls.c

```diff
@@ -43,7 +42,7 @@ typedef enum TLSIO_STATE_ENUM_TAG
 
 typedef struct SEND_COMPLETE_INFO_TAG
 {
-    int send_complete_count;
+    int is_fragmented_req;
```

Comment:
Nit: you can use bool if needed. Stdbool is already included.

Reply:
updated to bool

---
In reply to: [354467808](https://github.com/Azure/azure-c-shared-utility/pull/421#discussion_r354467808) [](ancestors = 354467808)